server/utils/streams.js

```diff
@@ -0,0 +1,170 @@
+const fs = require('fs');
+const path = require('path');
+const promisify = require('util').promisify;
+
+const parser = require('nomnom');
+const through = require('through2');
+const split = require('split');
+const request = require('request');
+
+const readDir = promisify(fs.readdir);
+
+function run() {
+  const { action, file, path } = parser
+    .option('action', {
+      abbr: 'a',
+      help: 'Name of action',
+      position: 0,
+    })
+    .option('file', {
+      abbr: 'f',
+      help: 'File path'
+    })
+    .option('path', {
+      abbr: 'p',
+      help: 'Directory path'
+    })
+    .parse();
+
+  if (action) {
+    switch (action) {
+      case 'transform':
+        transform();
+        break;
+      case 'io':
+        inputOutput(file);
+        break;
+      case 'transform-file':
+        transformFile(file);
+        break;
+      case 'bundle-css':
+        bundle(path);
+        break;
+    }
+  } else {
+    console.log('Usage: node streams.js [action] [options]');
+  }
+}
+
+function help(param) {
```

Comment:
Please export all supplementary functions

Reply:
Also fixed, but this string was not changed :)